.github/workflows/e2e.yml

```diff
@@ -88,7 +112,7 @@ jobs:
                       ~/.cache/Cypress
                   key: ${{ runner.os }}-cypress-node-modules-3-${{ hashFiles('**/yarn.lock') }}
                   restore-keys: |
-                      ${{ runner.os }}-cypress-node-modules-3
+                      ${{ runner.os }}-cypress-node-modules-3-
```

Comment:
?

Reply:
Technically it should be a prefix, it's not that important, unless we end up with `${{ runner.os }}-cypress-node-modules-31-${{ hashFiles('**/yarn.lock') }}` as a key.